lib/scope.go

```diff
@@ -119,8 +119,8 @@ func (s *scope) AppContext() context.Context {
 }
 
 // Collection returns the Instance collection subsystem
-func (s *scope) CollectionSet() collection.Set {
-	return s.inst.collectionSet
+func (s *scope) CollectionSet() *collection.Collection {
```

Comment:
Method is named `Collection.Set` but returns a `collection.Collection` rather than a `collection.Set`.